client/branded/package.json

```diff
@@ -4,9 +4,9 @@
   "version": "1.0.0",
   "license": "Apache-2.0",
   "scripts": {
-    "storybook": "STORIES_GLOB='client/branded/src/**/*.story.tsx' yarn workspace @sourcegraph/storybook run start",
+    "storybook": "STORIES_GLOB='client/branded/src/**/*.story.tsx' pnpm -F @sourcegraph/storybook start",
     "lint:js": "eslint --cache '**/*.[jt]s?(x)'",
     "lint:css": "stylelint 'src/**/*.scss' --quiet",
-    "test": "yarn run -T jest"
```

Comment:
Ok, all these `-T` calls... https://yarnpkg.com/cli/run#options

```
-T, --top-level

Check the root workspace for scripts and/or binaries instead of the current one
```

So doing `yarn run -T` looks for `jest` in the root `node_modules/.bin` **or** the `scripts` in the root package.json. That **or** is what makes it awkward imo. I think we don't need the `-T` when executing a binary, but we do when executing a script...

Reply:
Note that it keeps the same cwd, just allows running `scripts` from the root package.json.

So 90% of the `yarn run -T` can just be dropped because pnpm puts the parent `node_modules/.bin` on the PATH, it's only when running `scripts` that we need to do something with pnpm...